src/routes/+page.svelte

```diff
@@ -1,3 +1,88 @@
-<main class="flex h-full items-center justify-center">
-	<h1 class="w-fit rounded-lg bg-secondary-400 p-4 text-white">Your work goes on this page</h1>
+<script>
+	import Feature from './feature.svelte';
+</script>
+
+<main class="flex h-full items-center justify-center bg-surface-900">
+	<Feature
+		title="Optimize Your Cloud Spend with Granular Cost Management"
+		description="Gain complete visibility and control over your Kubernetes costs with our comprehensive cost management features. Our platform provides you with the tools you need to:"
+		ctaText="Book a free demo"
+		ctaLink="#"
+		imageSrc="/cost-management-image.png"
+		imageAlt="Cost Management Feature"
+		><slot>
```

Comment:
It's not necessary to wrap the markup inside of a `<slot>` tag here. You can safely remove the `slot`

Reply:
Oh, thanks